src/componenten/buttons/resetButton.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function resetButton(){
```

Comment:
React componenten horen altijd met een hoofdletter te beginnen
```suggestion
function ResetButton(){
```